client/src/components/Home.tsx

```diff
@@ -204,14 +204,27 @@ function Home() {
       {user && <Releases />}
       {user && posts.length > 0 && (
         <>
-          <h2
+          <h1
```

Comment:
This should be `h2` imo. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page

Reply:
No problem, I'll have to refactor most header behaviors at some point to be more consistent across pages (I've started doing that already and this was part of it but I'll change what I've done with H1 to function with H2)